docs/COMPILE.md

```diff
@@ -296,3 +298,61 @@ Signed-By: /usr/share/keyrings/box64-archive-keyring.gpg" | sudo tee /etc/apt/so
 sudo apt update
 sudo apt install box64-generic-arm -y
 ```
+
+## Cross-compiling 
+----
+### Set Up the Cross-Compiler
+For example, to compile Box64 for RISC-V on an x86 machine, you can select the latest or a specialized toolchain from the [riscv-gnu-toolchain](https://github.com/riscv-collab/riscv-gnu-toolchain/releases) 
+
+### Run CMake with Cross-Compilation Options 
+Follow the per-platform compilation instructions to configure the CMake options for your target architecture.
+In particular, you must specify the cross-compiler. For example:
+```
+-DCMAKE_C_COMPILER=riscv64-unknown-linux-gnu-gcc  # Or whichever cross-compiler you use
+```
+
+### Running tests with QEMU (optional)
+If you have QEMU’s RISC-V user-mode emulator (e.g., qemu-riscv64) installed on your x86 host, you can verify that your cross-compiled Box64 binary works correctly. To do a quick check, run:
+```
+qemu-riscv64 -L path/to/your/riscv64/sysroot box64 --help
+```
+Here, path/to/your/riscv64/sysroot should contain the necessary libraries and binaries for the RISC-V environment so that QEMU can correctly emulate it.
+
+#### Running CTest with QEMU
+To run CTest-based tests under QEMU, set the required environment variables and run ctest. Below is an example for RISC-V:
+```
+# Define Basic CTest Options

```

Comment:
Hmmm, this is way to complicated. It's important to test various configurations in the CI, but in documentation, just `ctest -j$(nproc)` should be good enough...